app.rb

```diff
@@ -4,119 +4,19 @@
 require_relative 'teacher'
 require_relative 'rental'
 require_relative 'classroom'
+require_relative 'display_module'
+require_relative 'create_instances_module'
+require_relative 'menu_handler_module'
 
 class App
+  include DisplayModule
+  include CreateInstancesModule
+  include MenuHandlerModule
   attr_accessor :books, :people, :rentals
 
   def initialize
     @books = []
     @people = []
     @rentals = []
   end
-
```

Comment:
- [ ] I was not able to run your app due to naming convention which i corrected, and got it working 👍 .
- Kindly look at the error prompt below, and make the appropriate changes 

### Error and rectified view:
![image](https://github.com/eamador1/school-library/assets/50990925/6ad6dba6-ea4d-41cd-962e-b0d935e8ac9d)